mpas_analysis/ocean/climatology_map_argo.py

```diff
@@ -0,0 +1,461 @@
+                             'list of depths'.format(sectionName))
+
+        # the variable 'timeMonthly_avg_landIceFreshwaterFlux' will be added to
+        # mpasClimatologyTask along with the seasons.
+        remapClimatologySubtask = RemapDepthSlicesSubtask(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName=fieldName,
+            variableList=[mpasFieldName],
+            seasons=seasons,
+            depths=depths,
+            comparisonGridNames=comparisonGridNames,
+            iselValues=iselValues)
+
+        if refConfig is None:
+
+            refTitleLabel = 'Roemmich-Gilson Argo Climatology: Temperature'
+
+            observationsDirectory = build_config_full_path(
+                config, 'oceanObservations', 'argoSubdirectory')
+
+            obsFileName = \
+                '{}/ArgoClimatology_TS.nc'.format(
+                        observationsDirectory)
+            refFieldName = 'theta'
+            outFileLabel = 'tempArgo'
+            galleryName = 'Roemmich-Gilson Climatology: ARGO'
+            diffTitleLabel = 'Model - Argo'
+
+            remapObservationsSubtask = RemapArgoClimatology(
+                    parentTask=self, seasons=seasons, fileName=obsFileName,
+                    outFilePrefix=refFieldName,
+                    fieldName=refFieldName,
+                    depths=depths,
+                    comparisonGridNames=comparisonGridNames)
+
+            self.add_subtask(remapObservationsSubtask)
+
+        else:
+            remapObservationsSubtask = None
+            refRunName = refConfig.get('runs', 'mainRunName')
+            galleryName = 'Ref: {}'.format(refRunName)
+            refTitleLabel = galleryName
+
+            refFieldName = mpasFieldName
+            outFileLabel = 'temp'
+            diffTitleLabel = 'Main - Reference'
+
+        for comparisonGridName in comparisonGridNames:
+            for season in seasons:
+                for depth in depths:
+                    subtask = PlotClimatologyMapSubtask(
+                        parentTask=self,
+                        season=season,
+                        comparisonGridName=comparisonGridName,
+                        remapMpasClimatologySubtask=remapClimatologySubtask,
+                        remapObsClimatologySubtask=remapObservationsSubtask,
+                        refConfig=refConfig,
+                        depth=depth)
+
+                    subtask.set_plot_info(
+                        outFileLabel=outFileLabel,
+                        fieldNameInTitle='Temperature',
+                        mpasFieldName=mpasFieldName,
+                        refFieldName=refFieldName,
+                        refTitleLabel=refTitleLabel,
+                        diffTitleLabel=diffTitleLabel,
+                        unitsLabel=r'$^\circ$C',
+                        imageCaption='ARGO_Temperature',
```

Comment:
The image caption should describe the figure you're seeing when you have a single image up in the browser.  Could you make this a bit more descriptive? Something like "Temperature compared with ARGO observations"?

Reply:
sure will do.  A question -- where does the imageCaption appear?  I don't see ARGO_temperature in the figures I generate.